src/components/HeaderApp.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="wrapper-header">
+      <div class="header">
+        <router-link class="header__logo" to="/">
+          <img class="header__logo-img" src="@/assets/logo.svg" alt="logo">
+        </router-link>
+      <div class="header__nav">
+        <v-btn to="/raiting-movies" x-small class="header__nav__btn">Raiting films</v-btn>
+        <v-btn to="/some-movies" x-small class="header__nav__btn">Some films</v-btn>
+      </div>
+    </div>
+    </div>
+</template>
+
+<script>
+export default {
+  name: 'HeaderApp'
```

Comment:
HeaderApp naming. It sounds like this is a separate Header application.
Please, rename this component to ex: "AppHeader" or "SharedHeader"

Reply:
Done